src/iarc7_motion/iarc_tasks/height_holder.py

```diff
@@ -30,13 +33,17 @@ def __init__(self):
     # that is set as the param _track_roomba_height
     def get_height_hold_response(self, height):
         with self._lock:
-            delta_z = self._TRACK_HEIGHT - height
-            response = self._K_Z * delta_z
+            if self._current_height is None:
+                self._current_height = height
+
+            if not self._USE_CURRENT_HEIGHT:
+                self._delta_z = self._TRACK_HEIGHT - height
```

Comment:
Additionally, the way this works now, get_height_hold_response does two different things without really indicating whats going on.